### PR TITLE
2408 welcome banner

### LINKED ISSUE #2408: Update homepage welcome banner
Update the homepage 'welcome banner' for unauthenticated first time users.

---

As per Ned's directive, let's update the banner.

**Mobile layout**

screen shot 2018-01-25 at 6 55 37 pm

**Desktop layout**

screen shot 2018-01-25 at 6 55 42 pm

**Details** 

Headline copy: `Your voice is worth something`

Body copy: `Get paid for good content. Post and upvote articles on Steemit to get your share of the daily rewards pool.`

The 'learn more' button should link to https://steemit.com/faq.html (Yes, I know, it's not the loveliest page on the site but it provides useful info – hopefully we'll improve it soon).

**Hero image**

I've attached a zip file with the hero image in .webp and .png format. To keep this performant as possible, please use a technique like below to serve the smaller sized webp image to capable browsers (mainly Chrome for now but that represents most of our traffic).

``

[hero-image.zip](https://github.com/steemit/condenser/files/1666468/hero-image.zip)

### PR DESCRIPTION
**Pull request by relativityboy**
Closes #2408

### CODE REVIEW

**Comment by bnchdrff** [comment]
a 95% quality jpg is 90k vs the 200k png or 57k webp

**Comment by bnchdrff** [comment]
ah we need alpha though. 200k is fine really.

**Comment by bnchdrff** [comment]
when i test locally, the css animations happen twice: once at initial render and again after DOMContentLoaded -- is this intended?

**Comment by gl2748** [comment]
Tested. LGTM.
One request would be to rebase and squash.

## Demo.

overlay

**Comment by relativityboy** [comment]
@bnchdrff - the video you posted suggests you don't have latest, or that the server wasn't restarted after doing a pull. 
Why I think this - The welcome banner doesn't ssr.

File: src/app/components/App.jsx [hunk 1]
```diff
@@ -33,6 +33,64 @@ const pageRequiresEntropy = path => {
     return entropyPages.indexOf(page) !== -1;
 };
 
+class WelcomePanel extends React.Component {
```
Comment (bnchdrff) [inline src/app/components/App.jsx]: this should live in its own file

Comment (relativityboy) [reply src/app/components/App.jsx]: I agree in principle, as it's a private component (not used elsewhere, not exported) I'm ok with it here until there's more bandwidth.

**Pull request merged**